### PR TITLE
Simplify leftover `Function` imports

### PR DESCRIPTION
**Pull request by Saransh-cpp**
Simplified every `Function` import.

`Function` was at the 3rd position of "dot-analysis" and now it is not anywhere in the top 10.

Also somewhat related to #759

### CODE REVIEW

File: README/Data/Container/Indexed.agda [hunk 1]
```diff
@@ -12,7 +12,7 @@ open import Data.Unit
 open import Data.Empty
 open import Data.Nat.Base
 open import Data.Product
-open import Function
+open import Function using (_∋_)
```
Comment (gallais) [inline README/Data/Container/Indexed.agda:15]:
```suggestion
open import Function.Base using (_∋_)
```

File: src/Data/Graph/Acyclic.agda [hunk 2]
```diff
@@ -26,7 +26,7 @@ open import Data.Empty
 open import Data.Unit.Base using (⊤; tt)
 open import Data.Vec.Base as Vec using (Vec; []; _∷_)
 open import Data.List.Base as List using (List; []; _∷_)
-open import Function
+open import Function using (_$_; _∘′_; _∘_; id)
```
Comment (gallais) [inline src/Data/Graph/Acyclic.agda:29]:
```suggestion
open import Function.Base using (_$_; _∘′_; _∘_; id)
```

**Review by gallais — COMMENTED** [review]
Nice!

File: README/Data/Container/Indexed.agda (hunk 1 above)
Comment (Saransh-cpp) [reply README/Data/Container/Indexed.agda:15]: Oops, missed these. Thanks!

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**